types/ThreadQueryParams.ts

```diff
@@ -43,11 +46,13 @@ export interface TimelineViewMode {
   timelineViewMode: TIMELINE_VIEW_SUB_MODE;
 }
 
+// TODO: rename this to BaseViewMode
```

Comment:
Maybe also TODO rename the `threadViewMode` properties in the `ClassicThreadViewMode`, `GalleryViewMode`, and `TimelineViewMode` interfaces above to `baseViewMode`